sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubUnboundedSink.java

```diff
@@ -495,6 +604,10 @@ static class PubsubSink extends PTransform<PCollection<byte[]>, PDone> {
       this.outer = outer;
     }
 
+    boolean isDynamic() {
```

Comment:
remove? seems PubsubUnboundedSink would be used if this was true?

Reply:
done